src/include/general.h

```diff
@@ -128,4 +130,43 @@ void debug_serial_send_stdout(const uint8_t *data, size_t len);
 #undef MAX
 #define MAX(x, y) (((x) > (y)) ? (x) : (y))
 
+#if defined(_WIN32) && !defined(__clang__)
+#define UNUSED
+#else
+#define UNUSED __attribute__((unused))
+#endif
+
+#ifdef _MSC_VER
+#define strcasecmp _stricmp
+static inline int strncasecmp(const char* const a, const char* const b, size_t size)
+{
+	if (a == b)
+		return 0;
+
+	char *c = malloc(size * sizeof(char) + 1);
```

Comment:
Is it possible to use `_strnicmp` here to avoid allocating? If not, please check the result of these allocations as this is liable to strncpy_s() to a `NULL` pointer otherwise.

Reply:
It seems like it, I couldn't find this function previously. Thanks!